core/src/main/resources/jenkins/model/Jenkins/oops.jelly

```diff
@@ -32,7 +32,7 @@ THE SOFTWARE.
     <l:header />
     <l:main-panel>
         <h1 style="text-align: center">
-          <img src="${imagesURL}/rage.svg" height="179" width="154"/> <span style="font-size:50px"><st:nbsp/>${%Oops!}</span>
+          <img src="${imagesURL}/rage.svg" height="179" width="154" alt=""/> <span style="font-size:50px"><st:nbsp/>${%Oops!}</span>
```

Comment:
Inconsistent with the change in `_404_simple.jelly`, why?

(Personally I like this one more for the same reason as I mentioned in a number of other comments.)

Reply:
Resolved in 23d56f1cf8928bbe8388f4c0fde302779ee594e2 by using empty alt text

Reply:
Not true, this is the only line in this PR you didn't update 😉

Reply:
Thanks for catching that.  This was resolved in 4d8fd426116664abdb7d45af29ba2e226821b76d